geocity/apps/submissions/models.py

```diff
@@ -1142,6 +1142,24 @@ def get_history(self):
         history.sort(reverse=True)
         return history
 
+    @property
+    def sent_date(self):
+        return self.get_sent_date()
+
+    def get_sent_date(self):
+        history = self.get_history()
+        sent_date = None
+
+        for date, archive in history:
+            if archive.status == Submission.STATUS_SUBMITTED_FOR_VALIDATION:
+                sent_date = archive.history_date
+
+                # Since the history is reverse sorted, we only want the first entry
+                # with status = STATUS_SUBMITTED_FOR_VALIDATION
+                break
+
+        return sent_date
```

Comment:
Please avoid loops in api. But I don't know if django-simple-history supports additional filters ?